docfx_yaml/extension.py

```diff
@@ -1235,7 +1304,8 @@ def convert_module_to_package_if_needed(obj):
                   'uid': uid
                 })
 
-    if len(toc_yaml) == 0:
+    # Exit if there are no generated YAML pages or Markdown pages.
+    if len(toc_yaml) == 0 and len(app.env.markdown_pages) == 0:
```

Comment:
Feels funny to call it `toc_yaml` if we end up adding more to it. Perhaps we should rename `toc_yaml` to `pkg_toc_yaml` or something to make it clear it's not the complete `toc`?

Reply:
Sounds good. Done.